exams/pearson/readers.py

```diff
@@ -54,6 +54,27 @@ def parse_datetime(cls, dt):
         """
         return datetime.strptime(dt, PEARSON_DATETIME_FORMAT).replace(tzinfo=pytz.UTC)
 
+    @classmethod
+    def map_cell(cls, row, source, target, transformer=None):
```

Comment:
`self` isn't used here so there's no need for it to be an instance method and have to add `# pylint: disable=no-self-use` clutter